packages/realm/bindgen/vendor/bindgen-lib/.eslintrc.json

```diff
@@ -1,7 +1,10 @@
 {
+  "root": true,
   "parser": "@typescript-eslint/parser",
   "extends": [
-    "plugin:@typescript-eslint/recommended"
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended"
```

Comment:
These should probably be added as dependencies to the package as well, so it doesn't rely on a parent package providing them.

Reply:
I'm going to do a pass on that package.json to make sure it really declares everything it needs as part of moving it to the core repo. I'll defer this until then, since I'm sure ill miss some stuff.